src/resource/resource.ts

```diff
@@ -29,13 +29,17 @@ import {
   WithResourceInfo,
   hasServerResourceInfo,
   SolidClientError,
+  SolidDataset,
 } from "../interfaces";
 import { internal_toIriString } from "../interfaces.internal";
 import { fetch } from "../fetcher";
 import {
   internal_isUnsuccessfulResponse,
   internal_parseResourceInfo,
 } from "./resource.internal";
+import { getThing } from "../thing/thing";
+import { getIriAll } from "../thing/get";
+import { ldp } from "rdf-namespaces";
```

Comment:
This is only a devDependency for convenience in the test, but we don't want to add it to our bundle just for the couple of URLs we use. In lieu of #114, we're just keeping them in `src/constants.ts`. (Though it looks like you're not using this import in the first place, so removing it is even safer :) )

```suggestion
```